.pre-commit-config.yaml

```diff
@@ -1,9 +1,25 @@
+exclude: ^tin/static/.*vendor
```

Comment:
Have you looked at the excluded files [here](https://github.com/tjcsl/tin/blob/master/pyproject.toml#L5-L17)? Some of them seem relevant/important to include here, but I'm not sure if that makes a difference.

Reply:
I just circumvented the issue by only checking `^tin/.*` when linting/formatting. The only id without that exception is `codespell`, which I _think_ should be fine, since codespell is limited to `^.*\.(py|md|rst)`